apps/mull-ui-e2e/src/integration/US-1.1.spec.ts

```diff
@@ -78,8 +103,16 @@ frameSizes.forEach((frame) => {
 
       cy.get('#description').type('test description');
 
-      cy.get('#location').type('test location');
+      cy.get('#location').click();
+      cy.get('#location-input-field').should('be.visible');
+
+      cy.get('#location-input-field').type('845 Rue Sherbrooke');
+      cy.get('#location-input-field-option-0', { timeout: 3500 }).should(
+        'have.text',
+        '845 Rue Sherbrooke, Montreal, QC, Canada'
+      );
```

Comment:
Same issue as above here
```suggestion
      cy.get('#location-input-field-option-0', { timeout: 3500 }).should(
        'contain.text',
        '845 Rue Sherbrooke'
      );
```